frontend/src/lib/components/Cards/TextCard/TextCard.scss

```diff
@@ -0,0 +1,53 @@
+.TextCard {
+    z-index: 3;
+    background: #fff;
+}
+
+.TextCard-Body {
+    ul {
+        list-style: disc;
+        padding-inline-start: 1.5em;
+    }
+
+    ol {
+        list-style: numeric;
+        padding-inline-start: 1.5em;
+    }
+
+    p {
+        margin-bottom: 0;
+    }
+}
+
+.handle {
```

Comment:
Why's this duplicated from `InsightCard.scss`? This way there are two conflicting class declarations:
<img width="292" alt="Screen Shot 2022-10-11 at 17 51 55" src="https://user-images.githubusercontent.com/4550621/195140021-7038f2e9-a722-4894-b476-b48632c87e98.png">